docker/tap-schema/1015_dp01_dc2_tap_schema.sql

```diff
@@ -0,0 +1,1185 @@
+INSERT INTO tap_schema.schemas11 (schema_name, utype, description, schema_index) VALUES ('dp01_dc2_catalogs.dp01_dc2_catalogs', null, 'Data Preview 0 includes five tables based on the DESC’S Data Challenge 2 simulation of 300 square degrees of the wide-fast-deep LSST survey region after 5 years. All tables contain objects detected in coadded images.', 0);
```

Comment:
Shouldn't the schema name be just `dp01_dc2_catalogs`, without the `.` and repetition?  This would apply to all the `INSERT INTO tap_schema.tables11` commands as well.  Is this a problem with the Felis-to-TAP_SCHEMA toolchain?